src/test/java/kappzzang/jeongsan/service/GetTransferTargetTest.java

```diff
@@ -0,0 +1,125 @@
+import kappzzang.jeongsan.domain.Member;
+import kappzzang.jeongsan.domain.PersonalExpense;
+import kappzzang.jeongsan.domain.Team;
+import kappzzang.jeongsan.dto.request.TransferTargetRequest;
+import kappzzang.jeongsan.dto.request.TransferTargetRequest.ExpenseId;
+import kappzzang.jeongsan.dto.response.TransferTargetResponse;
+import kappzzang.jeongsan.repository.MemberRepository;
+import kappzzang.jeongsan.repository.PersonalExpenseRepository;
+import kappzzang.jeongsan.repository.TeamRepository;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+public class GetTransferTargetTest {
+
+    Member member1, member2, member3, member4, payer;
+    List<PersonalExpense> personalExpenses;
+    Map<Long, Integer> expectedTotalPrices;
+
+    @Mock
+    private TeamRepository teamRepository;
+    @Mock
+    private PersonalExpenseRepository personalExpenseRepository;
+    @Mock
+    private MemberRepository memberRepository;
+    @InjectMocks
+    private TeamService teamService;
+
+    @BeforeEach
+    void setUp() {
+        setUpMembers();
+        setUpPersonalExpenses();
+        setUpExpectedTotalPrices();
+    }
+
+    @Test
+    @DisplayName(
+        "request에 포함된 지출에 대해 멤버별로 지불해야 할 금액을 합하여 반환한다. "
+            + "이때, API 호출자는 결제자이므로 response에 포함하지 않는다.")
+    void getTransferTargetList() {
+        // given
+        given(memberRepository.findById(anyLong())).willReturn(Optional.ofNullable(payer));
+        given(teamRepository.findById(anyLong())).willReturn(Optional.of(new Team()));
+        given(personalExpenseRepository.findAllByExpenseIdsWithItemAndMember(anyList()))
+            .willReturn(personalExpenses);
+
+        ExpenseId id1 = new ExpenseId(1L);
+        ExpenseId id2 = new ExpenseId(1L);
+
+        TransferTargetRequest request = new TransferTargetRequest(List.of(id1, id2));
+
+        // when
+        List<TransferTargetResponse> response =
+            teamService.getTransferTargetList(5L, 1L, request);
+
+        // then
+        for (TransferTargetResponse targetResponse : response) {
+            Long memberId = targetResponse.memberId();
+            Integer actualTotalPrice = targetResponse.amountDue();
+            Integer expectedTotalPrice = expectedTotalPrices.get(memberId);
+
+            assertNotEquals(payer.getId(), memberId);
+            assertEquals(expectedTotalPrice, actualTotalPrice);
+        }
+    }
+
+    private void setUpMembers() {
+        member1 = new Member(1L, "member1", "member1");
+        member2 = new Member(2L, "member2", "member2");
+        member3 = new Member(3L, "member3", "member3");
+        member4 = new Member(4L, "member4", "member4");
+        payer = new Member(5L, "payer", "payer");
+    }
```

Comment:
P2: Member 엔티티 모킹을 통해 id 필드 생성자 없이도 테스트가 가능할것 같습니다!
```suggestion
 private void setUpMembers() {                                                         
     for (int i = 0; i < 4; i++) {                                                    
         given(members.get(i).getId()).willReturn((long) i);                          
         given(members.get(i).getNickname()).willReturn("member" + (i + 1));          
         given(members.get(i).getProfileImage()).willReturn("member" + (i + 1));      
     }                                                                                
     given(payer.getId()).willReturn(5L);                                             
 }                                                                                    
```

Reply:
엔티티 모킹 좋은 것 같습니다!